program_postgres.sql

```diff
@@ -95,10 +95,10 @@ INSERT INTO "public"."configuration" ("key", "value", "status") VALUES ('smsCont
+INSERT INTO sunbird.email_template (name, template)
+VALUES ('emailNominationReject', '<!doctype html><head> <meta name="viewport" content="width=device-width"> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <title></title> <style>@media screen and (min-width: 320px){.container{width: 100%; font-family: sans-serif; font-size: 14px; display: block; Margin: 0 auto; background: #fff; max-width: initial; padding: 25px;}body{background: #fff !important;}td, p{font-size: 15px !important;}}@media screen and (min-width: 767px){.container{max-width: 580px; padding: 8px 25px; width: 580px !important; margin-top: 25px !important;}body{background-color: #f6f6f6 !important;}}</style></head><body style="background-color: #f6f6f6; font-family: sans-serif; -webkit-font-smoothing: antialiased; font-size: 14px; line-height: 1.7; margin: 0; padding: 0; -ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%;"> <div class="container"> <table border="0" cellpadding="0" cellspacing="0" style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%;"> <tr> <td><img src="$baseUrl/tenant/ntp/logo.png" alt="logo" align="right" width="180" max-width="100%" style="padding-top: 15px;"> </td></tr><tr> <td style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin-bottom: 15px;line-height: 1.7;padding:10px 0"> Hi,</td></tr><tr> <td> <p style="font-family: sans-serif; font-size: 14px; font-weight: normal;margin-bottom: 15px;line-height: 1.7;"> This is to inform you that your nomination for the project $projectName has not been accepted. Please login to <a href="$baseUrl/" target="_blank">$baseUrl/</a> for details. </p></td></tr><tr> <td> <p style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin-bottom: 15px;line-height: 1.7;"> Thank you for your interest. You can nominate yourself / your organisation for other projects that are currently open for nomination, or for future projects. </p></td></tr><tr> <td> <div style="font-family: sans-serif; font-size: 14px; font-weight: normal;line-height: 1.7;"> Regards, </div><div style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin-bottom: 15px;line-height: 1.7;"> VidyaDaan Team </div></td></tr><tr> <td> <p style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 15px;line-height: 1.7;"> Note: This is an automatically generated email. Replies to this mailbox will not be monitored. If you are not the intended recipient of this message, or need to communicate with the team, write to <a href="mailto:vdnsupport@teamdiksha.org" target="_blank">vdnsupport@teamdiksha.org</a> </p></td></tr></table> </div></body></html>')
```

Comment:
@niteshkesarkar I think we will need to have update table queries. @pallakartheekreddy Please check

Reply:
also, this PR is raised for /pallakartheekreddy repo. I think that should be on Sunbird-Ed/